src/EditorFeatures/CSharpTest/Workspaces/WorkspaceTests.cs

```diff
@@ -1031,5 +1031,27 @@ public async Task TestLinkedFilesStayInSync()
                 Assert.Equal(updatedText, (await eventArgs[1].NewSolution.GetDocument(originalDocumentId).GetTextAsync().ConfigureAwait(false)).ToString());
             }
         }
+
+        [Fact, WorkItem(31928, "https://github.com/dotnet/roslyn/issues/31928")]
+        public void TestVersionStamp_Local()
+        {
+            // only Utc is allowed
+            Assert.Throws<ArgumentException>(() => VersionStamp.Create(DateTime.Now));
```

Comment:
won't this *not* throw if the user is actually running on a machine in utc?  or is DateTime.Now *never* utc?

Reply:
Now always return Local